### ISSUE TITLE
Status of the project

### ISSUE THREAD

**Issue by claudyus** [genropy/genropy#5]
Hi,
I'm start to evaluate genropy for an web application after your presentation at PyCon7. Anyway some doubts need to be address before this could be consider, in my opinion, an usable open source project.

Major problems:
- Licence is unclear. Multiple files are reporting non compatible licences
- No tags are present in repo also when commit message is 'new release', no semver
- The installation procedure say to use ```sudo``` without explain the motivation
- Under no condition "no message" can be consider an useful commit message
- Issues are reported in italian
- The issues discussion in not keep in public

Minor issues:
- commits are huge

Do you have any interest in addressing those problems or we should consider genropy an as-is tool?

**Comment by fporcari** [comment]
Our main repository is on bitbucket https://bitbucket.org/genropy/. 
github is used as just a mirror of stable version so this is the reason why commit here are huge.

Please refer to bitbucket as main version and www.genropy.org for any documentation info

> Do you have any interest in addressing those problems or we should consider genropy an as-is tool?
We are trying to improve Genropy project to meet OpenSource standards so any help is welcome 

We can discuss about it in the mailing list 
genropy-request@freelists.org

**Comment by claudyus** [comment]
I could suggest you to create a git mirror instread that randomly push a huge unrelated commit from one repo to another. 
This is very simple in git using multple remote, here and example: 
http://stackoverflow.com/questions/21551929/how-to-make-a-github-mirror-to-bitbucket

You can also sync subset of tags or branches if you don't want a full mirroring.

My 2 cents

**Comment by fporcari** [comment]
> You can also sync subset of tags or branches if you don't want a full mirroring.

Good point. In the next few weeks we are going to clean some parts of the code (comment, and removing old modules) in order to sync these repo and make it more friendly to new users

> My 2 cents

Isn't a free framework enough ;)

**Comment by fedelibre** [comment]
@fporcari genropy.org is 3rd on google search result, BUT there's a weird description which I immediatley classified as spam so I did not click on it until I found this github page:

![genropy-search](https://cloud.githubusercontent.com/assets/1644647/21354838/db889eb4-c6cb-11e6-8d93-49f8b40d33eb.png)

**Comment by bittner** [comment]
This issue, I think, is still acute. Strange also that the [Genropy website](https://www.genropy.org) explicitly states on top of the page that the project is hosted on GitHub.

**Comment by bittner** [comment]
I've opened an [issue on Bitbucket](https://bitbucket.org/genropy/genropy/issues/1/configure-mirror-automatic-sync-to-github) for configuring mirroring.

**Comment by genro** [comment]
> Il giorno 24 apr 2018, alle ore 09:20, Peter Bittner <notifications@github.com> ha scritto:
> 
> I've opened an issue on Bitbucket for directing people to the Bitbucket repo.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 

Hi Peter
we are currently away for some days but we will work on it in a few days.

Thank you

**Comment by deepserket** [comment]
@bittner is right, when i saw this github project i thought it was dead (last commit 1 year ago, 1 issue, 0 pull request...), fortunately i was curious about this issue, but I think other people may have abandoned the idea of ​​using this framework after having looked at the github.
And please add the link to bitbucket in genropy.org

**Comment by fporcari** [comment]
@bittner @deepserket I've just change the link inside genropy.org. Now it refers to the bitbucket repo. We actually suggest 2 branches develop for testing the newest features and master in production deployment. Thank you, every suggestions is really appreciated
